### ISSUE TITLE
Download links not working

### ISSUE THREAD

**Issue by xxxxvelocityxxxx** [niuzhiheng/caffe#74]
Hi. Your download links for the content you have on dropbox aren't working. Is there any way you could post a mirror link? Thanks.

**Comment by shthed** [comment]
Yes please!  The download link to the [standalone package](https://dl.dropboxusercontent.com/u/3466743/caffe-vs2012/standalone.7z) gives a DropBox Error (429)

> This account's public links are generating too much traffic and have been temporarily disabled!

**Comment by IZEDx** [comment]
So does the download for the Dependencies.

**Comment by ShineeSheng** [comment]
Hi , I cannot open the Dependecies,do you have the link?Thanks.

**Comment by sunshinenum** [comment]
So do I.
